Sources/TuistDependencies/DependenciesController.swift

```diff
@@ -0,0 +1,59 @@
+import TSCBasic
+import TuistSupport
+
+// MARK: - Dependencies Controller Errors
+
+public enum DependenciesControllerError: FatalError {
+    case unimplemented
+
+    /// Error type.
+    public var type: ErrorType {
+        switch self {
+        case .unimplemented:
+            return .abort
+        }
+    }
+
+    /// Description.
+    public var description: String {
+        switch self {
+        case .unimplemented:
+            return "Pssst! You have found secret and hidden part of project where we are trying to create something new and unexpected. Chimera!"
```

Comment:
Nitpick: I'd rephrase this to:
```
A standard approach for managing third-party dependencies is being worked on and it'll be available soon.
```

Reply:
I wanted to be funny here :P Im replacing with your suggestion :)